PySDM/backends/impl_thrust_rtc/storage.py

```diff
@@ -419,6 +424,9 @@ def _to_host(self):
         def amin(self):
             return Impl.amin(self.data)
 
+        def amax(self):
```

Comment:
perhaps, let's add a minimal unit test for this addition

Reply:
added